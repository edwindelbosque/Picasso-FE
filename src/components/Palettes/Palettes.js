import React from 'react';
import './Palettes.scss';
import PaletteCard from '../PaletteCard/PaletteCard';

const Palettes = ({ menuIsActive, palettes }) => {
	return (
		<section className='Palettes'>
			<h2>Palettes</h2>
			{palettes.map(palette => {
				const { id } = palette;
				return (
					<PaletteCard key={id} menuIsActive={menuIsActive} palette={palette} />
				);
			})}
		</section>
	);
};

export default Palettes;
