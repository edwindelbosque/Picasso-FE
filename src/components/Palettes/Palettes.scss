.Palettes {
	margin-right: 100px;
	padding-top: 40px;
	margin-top: 60px;
	padding-bottom: 100px;
	overflow: scroll;
	width: 750px;
	height: 92%;
	display: flex;
	flex-direction: column;
	align-items: center;
	background: rgba(255, 255, 255, 0.932);
	// border-left: 3px solid white;

	-moz-box-shadow: inset 0 0 15px #00000046;
	-webkit-box-shadow: inset 0 0 15px #0000005e;
	box-shadow: inset 0 0 15px #00000052;

	a {
		text-decoration: none;
		color: black;
	}

	h2 {
		color: rgb(48, 48, 48);
		width: 635px;
	}
}
