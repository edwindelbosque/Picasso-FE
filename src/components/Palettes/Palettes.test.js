import React from 'react';
import Palettes from './Palettes';
import { shallow } from 'enzyme';

describe('Palettes', () => {
	let wrapper;

	beforeEach(() => {
		wrapper = shallow(<Palettes />);
	});

	it('App should match snapshot', () => {
		expect(wrapper).toMatchSnapshot();
	});
});
