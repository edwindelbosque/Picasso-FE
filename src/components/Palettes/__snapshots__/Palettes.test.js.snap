// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Palettes App should match snapshot 1`] = `
<h1>
  Palettes
</h1>
`;
